.github/workflows/main.yml

```
@@ -54,20 +54,45 @@ jobs:
           asset_name: hbase-table-provisioner-${{ needs.get-publish-version.outputs.publish-version }}.jar
           asset_content_type: application
 
-  build-app:
+#  build-app:
+#    runs-on: ubuntu-latest
+#    steps:
+#      - uses: actions/checkout@master
+#      - name: build and test
+#        run: make integration-all
+#      - name: java setup
+#        uses: actions/setup-java@v1
+#        with:
+#          java-version: 1.8
+#      - name: gradle build
+#        run: make local-all
+#      - name: Upload app
+#        uses: actions/upload-artifact@v2
+#        with:
+#          name: built-app
+#          path: build/libs
+
+  publish-docker:
     runs-on: ubuntu-latest
+#    needs: [get-publish-version, build-app]
+    needs: [get-publish-version]
```

```suggestion
    needs: get-publish-version
```
It's a bit weird when there's only one item it needs to not be in `[]`